### ISSUE TITLE
Cordova Local Notifications doesn't compile

### ISSUE THREAD

**Issue by olieidel** [BetterArguruments/nofapp#7]
We will be waiting for a fix. See their Issues [#426](https://github.com/katzer/cordova-plugin-local-notifications/issues/426), [#567](https://github.com/katzer/cordova-plugin-local-notifications/issues/567) and [#584](https://github.com/katzer/cordova-plugin-local-notifications/issues/584)

**Comment by stefanmdt** [comment]
any news on that?

**Comment by olieidel** [comment]
Nope. The most viable fix at the moment probably would be tubaman's fix of version 0.7.8 as seen here: [#584](https://github.com/katzer/cordova-plugin-local-notifications/issues/584)
